azure-pipelines.yml

```diff
@@ -18,7 +18,7 @@ steps:
   inputs:
     targetType: 'inline'
     script: |
-      sbt test
+      sbt +test
```

Comment:
Hva er forskjellen med og uten +?

Reply:
Når vi setter opp `crossScalaVersions` i build.sbt så kan du putte `+` foran en kommando i sbt for å kjøre kommandoen for alle scala-versjonene. `+test` vil derfor kjøre testene både på 2.13.6 og 3.0.1 her

Reply:
Kult 😄